src/app/organizations/organizations/organizations-paginator.directive.ts

```diff
@@ -0,0 +1,38 @@
+import { Directive, OnInit, ViewChild } from '@angular/core';
```

Comment:
This file is used to create something similar to the functionalities in `search-entry-table.ts` I tried to have `organizations.component.ts` to extend from this, but then there's this "No providers for OrganizationsPaginatorDirective
". 
**Right now this file is not being used**